components/mpas-ocean/src/shared/mpas_ocn_vel_tidal_potential.F

```diff
@@ -173,10 +175,10 @@ subroutine ocn_vel_tidal_potential_tend(ssh, surfacePressure, tend, err)!{{{
          kMax = maxLevelEdgeTop(iEdge)
 
          potentialGrad = - gravity*invdcEdge* &
-            (  tidalPotEta(cell2) - tidalPotEta(cell1) &
-             + (1.0_RKIND - ssh_sal_on) * betaSelfAttrLoad *( (ssh(cell2) - ssh(cell1)) & 
-                                                      + rho0gInv*(surfacePressure(cell2) - surfacePressure(cell1))) &
-             + ssh_sal_on * (ssh_sal(cell2) - ssh_sal(cell1)) )
+            (  nonSshTermsSplit * tidalPotEta(cell2) - tidalPotEta(cell1) &
+             + sshTermSplit * (1.0_RKIND - ssh_sal_on) * betaSelfAttrLoad * ((ssh(cell2) - ssh(cell1)) & 
+             + nonSshTermsSplit * rho0gInv*(surfacePressure(cell2) - surfacePressure(cell1))) &
+             + nonSshTermsSplit * ssh_sal_on * (ssh_sal(cell2) - ssh_sal(cell1)) )
```

Comment:
@gcapodag I discussed these lines with @philipwjones, and we decided that a separate pressure gradient option and separate loop is better than 1/0 coefficients for both clarity and performance.

Reply:
I agree, let's put an `if(config_time_integrator =='LTS')` before the`iEdge` loop you mentioned above, and also another one outside the `iEdge` loop in `src/shared/mpas_ocn_vel_tidal_potential.F`. If the time integration scheme is not LTS then we'll have the same loops, except with `sshTermSplit` and `nonSshTermsSplit` removed. Thanks for suggesting this change @mark-petersen @philipwjones !